worlds/stardew_valley/data/villagers_data.py

```diff
@@ -1,7 +1,7 @@
 from dataclasses import dataclass
 from typing import List, Tuple, Optional
 from .region_data import SVRegion
-from .mod_data import ModNames
+from worlds.stardew_valley.mods.mod_data import ModNames

```

Comment:
You might want to ask in the AP server, but I seem to remember someone telling me all imports should be relative, not absolute